README.md

```diff
@@ -39,7 +39,7 @@ angular.module('myApp', ['hljs']);
 ## Install with npm
 
 ```sh
-npm install highlight.js
```

Comment:
Hi @aa-neg, [highlight.js](https://www.npmjs.com/package/highlight.js) is the correct npm dependency.